src/components/item/form/DocumentForm.tsx

```diff
@@ -188,12 +286,19 @@ const DocumentForm = ({
 
   return (
     <Box id="document" display="flex" flexDirection="column" minHeight="0px">
-      <NameForm
-        setChanges={setChanges}
-        item={item}
-        required
-        updatedProperties={updatedProperties}
-      />
+      <Stack direction="row" spacing={2}>
+        <NameForm
+          setChanges={setChanges}
+          item={item}
+          required
+          updatedProperties={updatedProperties}
+        />
+        <DisplayNameForm
```

Comment:
Actually the display name is on autoFocus. I think it's the `name` field that should be focused by default.